app/views/layouts/openstudios.html.slim

```diff
@@ -12,10 +12,10 @@ html
       window['__env'] = "#{Rails.env}";
 
     = stylesheet_packs_with_chunks_tag 'application'
-    = javascript_packs_with_chunks_tag "application"
+    = javascript_packs_with_chunks_tag 'application'
 
   - classes = [@browser_as_class, controller_name, action_name].join(" ")
-  body class=classes ng-app="MauApp"
+  body class=classes id="openstudios__layout" ng-app="MauApp"
```

Comment:
Do we get this free with controller_name/action_name?  feels like we might should get `open_studios_domain__main` or something like that from the classes above.  If not, this is a fine add.

Reply:
also I know we're already mixing but maybe from here on we can use `open_studios` instead of `openstudios`

Reply:
That's fair. I'll name all the directories the same namespace that I'm using for the controllers/views `open_studios_subdomain`.